frontend/src/Screens/GlobalComponents/Navbar/Components/PageBoard/PageBoard.tsx

```diff
@@ -0,0 +1,13 @@
+import './PageBoard.css';
+
+function PageBoard() {
+  return (
+      <div>
```

Comment:
remove unused div

Reply:
I was removed the "PageBoard.tsx" convet to "src//Board/Board.tsx"